### PR TITLE
Deploy v1.3.0

### PR DESCRIPTION
**Pull request by erikquinn**
Various updates to nattrak, in preparation for a test ahead of CTP. Changes include a user interface refresh, conflict detection, improved track sorting options.

All work by Liesel Downes.

### CODE REVIEW

File: app/Services/VatsimDataService.php [hunk 1]
```diff
@@ -114,7 +124,10 @@ public function getActiveControllerAuthority(VatsimAccount $vatsimAccount)
     public function getActivePilotData(VatsimAccount $vatsimAccount)
     {
         $networkData = $this->getNetworkData();
-        if (! $networkData) return null;
+        if (! $networkData) {
+            return null;
+        }
+        $vatsimAccount->id = 899571;
```
Comment (williammck) [inline app/Services/VatsimDataService.php:130]: @liessdow is this still supposed to be set? looks like some debug override stuffs

**Review by williammck — COMMENTED** [review]
note to self: run migrations after deploy

**Comment by liessdow** [comment]
Definitely forgot to remove, so the line overriding cid needs to go. I'll
push a fix to the gander repo tmrw.

**Review by williammck — APPROVED** [review]

File: app/Services/VatsimDataService.php (hunk 1 above)
Comment (liessdow) [reply app/Services/VatsimDataService.php]: found time tonight, fixed https://github.com/vatsimnetwork/nattrakv1/pull/2/commits/198d5da3d23b79cc86ed6b7aaa8508822fd9a873

**Pull request merged**